index.html

```diff
@@ -1,9 +1,122 @@
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<head>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+<header>
+    <div>
+      <div>
+        <img class="blackHeart" src="https://raw.githubusercontent.com/rimboriri/lab-html-exercise/master/images/black-heart.png" alt="black heart" />
+        <span> Nifty Penguin Magic </span>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="#"> npm Enterprise </a></li>
+          <li><a href="#"> Products </a></li>
+          <li><a href="#"> Solutions </a></li>
+          <li><a href="#"> Resources </a></li>
+          <li><a href="#"> Docs </a></li>
+          <li><a href="#"> Support </a></li>
+        </ul>
+      </nav>
+    </div>
+  
+    <div class="search-bar-header">
+           <img class="npm-logo" src="https://github.com/rimboriri/lab-html-exercise/blob/master/images/npm-logo.png?raw=true" alt="npm-logo" /> 
+         
+  
+      
+       <button id="searchQuerySubmit" type="submit" name="searchQuerySubmit">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24"><path fill="#666666" d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" />
+        </svg>
+      </button>
+      
+     <input  type="text" placeholder="find packages" class="search-bar">
+      </input>
+       
+     <button type="search" class="search-button"> Search </button>
+     
+      <div>
+       <a href="#" class="nav-btn">Join</a>
+       <a href="#" class="nav-btn">Login</a>
+      </div>
+    </div>
+  </header>
+  
+  <section id="section1" class="sections1">
+      <h1>Build amazing things</h1>
+    
+    <article class="text">
+     <p>Essential JavaScript development tools that help you go to market faster and build powerful applications using
+       modern open source code.</p>
+    </article>
+    <div>
+       <a href="#" class="link-btn">See plans</a>
+       <a href="#" class="no-btn">Join for free</a>
+    </div>
+  </section>
+  
+  
+  <section id="section2">
+  <img class="hexagon" src="https://raw.githubusercontent.com/ironhack-labs/lab-html-exercise/3a5dda422dd6e6fc4444645556e2a1ecc05879e9/images/triangle-hexagon.svg" alt="hexagon" />
+      <h2>Bring the best of open source to your company</h2>
+  
+    <article class="text">
+      <p>npm is the tool used by
+  over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
+  Create an Org and get more out of the tools your team already knows and loves.</p>
+    </article>
+    </section>
+  
+  <section id="section3">
+   <div class="sec3">
```

Comment:
I feel section 3 is quite narrow compared to the example...
What do you reckon?